dm/tests/online_ddl/run.sh

```diff
@@ -29,6 +20,27 @@ function run() {
 	sed -i "/relay-binlog-name/i\relay-dir: $WORK_DIR/worker2/relay_log" $WORK_DIR/source2.yaml
 	dmctl_operate_source create $WORK_DIR/source1.yaml $SOURCE_ID1
 
+	run_sql_file $cur/data/check.db1.prepare.sql $MYSQL_HOST1 $MYSQL_PORT1 $MYSQL_PASSWORD1
+
+	cp $cur/conf/check-task.yaml $WORK_DIR/check-task.yaml
+
+	run_dm_ctl $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"check-task $WORK_DIR/check-task.yaml" \
+		"\"result\": true" 1
```

Comment:
what does this test check? i think maybe we should check the error from checker  matched with `your ddl is in pt/ghost online-ddl` 🤔